CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 1.9.6
+
+* Revert #597 (#639)
```

Comment:
Can we have a bit more detail on this one please? Someone reading this file not on GitHub, won't immediately have any sense of what was reverted.